packages/create-ripple-app/template/_tests/_common/test/e2e/integration/smoke/tide-is-up.feature

```diff
@@ -4,10 +4,10 @@ Feature: Test Tide backend is up
 
   @smoke
   Scenario: Route endpoint returns a correct response
-    Given a request is made to "/api/v1/route?site=<%= siteid %>&path=/"
+    Given a request is made to "/api/v1/route?site=4&path=/"
```

Comment:
Id `4` will not work for those site id is not same. And some independent site has no site id 4. Any solution to make it dynamic?

Reply:
adding <%= siteid %> will copy in the siteid per project on update - just make sure its set when you run the update.